src/ui/window.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <types.hpp>
+
+namespace ui_window {
```

Comment:
`nebula::windowing` or even without `windowing` [personal favourite] (in this case you'd have to change the names of the functions like `init` to `init_windowing`)